src/main/java/org/sonarlint/intellij/ui/vulnerabilities/TaintVulnerabilitiesPanel.kt

```diff
@@ -307,6 +313,39 @@ class TaintVulnerabilitiesPanel(private val project: Project) : SimpleToolWindow
         currentStatus?.let { populateTrees(it) }
     }
 
+    fun doesTaintExist(taintKey: String): LocalTaintVulnerability? {
```

Comment:
Reading the name I was expecting a boolean to be returned. Maybe something like `findIssueByKey`?
I think there is the same for "normal" issues